feature/gnoi/factory_reset/tests/factory_reset_test/factory_reset_test.go

```diff
@@ -0,0 +1,206 @@
+package factoryreset
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/testt"
+	frpb "github.com/openconfig/gnoi/factory_reset"
```

Comment:
Please run go fmt to fix the order: 

 	"github.com/openconfig/featureprofiles/internal/fptest"
+	frpb "github.com/openconfig/gnoi/factory_reset"
 	"github.com/openconfig/ondatra"
 	"github.com/openconfig/ondatra/gnmi"
 	"github.com/openconfig/testt"